Automattic/sensei
# Update @wordpress/scripts and JSDoc alignments

### Changes proposed in this Pull Request

* It updates the `@wordpress/scripts` in order to get the jsdoc alignment rule in sync with Gutenberg.

### Testing instructions


* Check JS functions with JSDoc, and make sure it continues validating the rule.
* Run a build, and make sure it continues working properly.
